Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM alpine:3.16.2 AS base
+
+RUN apk add --no-cache npm git openssh
+
+WORKDIR /home/node/
+
+COPY package.json package-lock-json* ./
```

Comment:
is there a benefit in first setting the work dir instead of directly copying in the file to the required location?